docs/examples.rst

```diff
@@ -2,18 +2,39 @@ Example Deployments
 ===================
 
 Deploying dask-jobqueue on different clusters requires a bit of customization.
-Below, we provide a few example deployments:
+Below, we provide a few examples from real deployments in the wild:
 
 
-Example PBS Deployment
-----------------------
+PBS Deployments
+---------------
 
 .. code-block:: python
 
    from dask_jobqueue import PBSCluster
 
-   cluster = PBSCluster(processes=6, threads=4, memory="16GB")
-   cluster.start_workers(10)
+   cluster = PBSCluster(queue='regular',
+                        project='DaskOnPBS',
+                        local_directory=os.getenv('TMPDIR', '/tmp'),
+                        threads=4,
+                        processes=6,
+                        memory='16GB',
+                        resource_spec='select=1:ncpus=24:mem=100GB')
 
-   from dask.distributed import Client
-   client = Client(cluster)
+   cluster = PBSCluster(processes=18,
+                        threads=4,
+                        memory="6GB",
+                        project='P48500028',
+                        queue='premium',
+                        resource_spec='select=1:ncpus=36:mem=109G',
+                        walltime='02:00:00',
+                        interface='ib0')
```

Comment:
One idea would be to point to the documentation for the machines that these work on. Users may find it useful to compare/contrast the configuration of individual systems. For example, this one came from: https://www2.cisl.ucar.edu/resources/computational-systems/cheyenne

Reply:
Agreed, do we have these names for the clusters defined here?

Reply:
I do not have such documentation online for our cluster... I can give a quick overview of our compute nodes if needed. Not sure if a small (relative to Cheyenne) french cluster will talk to many guys...

Reply:
this one is Cheyenne (link above). 

The slurm cluster came from @rsignell-usgs is USGS's Yeti Cluster. Maybe he can provide a working link to the cluster's documentation, I came up empty handed 😉 

@guillaumeeb - what is the name of you cluster? Is there online documentation?